script/utils/check-task-numbering.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -euo pipefail
+
+# This script checks that all task directories have a unique prefix, defined as a number
+# followed by a hyphen and optionally some text denoting the network before the number.
```

Comment:
Can you give an example, and what the upper and lower bounds of the file names could be?

Reply:
Outdated, but now that we are in `src-improvements` I've refactored so that we demand that all tasks are "nnn-<string>", unlike some old tasks that are "nnn.nnn-<string>" or "<network>-nnn-<string>".

If we run out of numbers or we really want to group tasks we can change it, but for now I would try this simpler approach.

Reply:
As for the file names, as long as you do "nnn-*", go wild.